workloads/api_load/common.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+source env.sh
+source ../../utils/common.sh
+source ../../utils/benchmark-operator.sh
+
+openshift_login
+
+export_defaults() {
+  network_type=$(oc get network cluster -o jsonpath='{.status.networkType}' | tr '[:upper:]' '[:lower:]')
+  export UUID=$(uuidgen)
```

Comment:
Normally when we run a test from this all `workloads` have the same UUID, se we can differentiate one run from another, it would be nice to be able to check if it has been defined. But maybe internally that `uuidgen` does that, have not gone to see yet.